scenario/sequences.py

```diff
@@ -58,9 +58,11 @@ def generate_startup_sequence(state_template: State):
         (
             (
                 Event(
-                    "leader_elected"
-                    if state_template.leader
-                    else "leader_settings_changed",
+                    (
+                        "leader_elected"
+                        if state_template.leader
+                        else "leader_settings_changed"
+                    ),
```

Comment:
This is not related to this PR, but `tox` was unhappy until I had `black` reformat this file.

Reply:
mm, perhaps a new release? looks ugly but okay :D